auth/middleware.go

```diff
@@ -35,6 +36,14 @@ func (k *kratosMiddleware) Session(next echo.HandlerFunc) echo.HandlerFunc {
 		if !*session.Active {
 			return c.String(http.StatusUnauthorized, "Unauthorized")
 		}
+
+		// Adding Authorization Token for PostgREST
+		token, err := k.generateDBToken(session.Identity.GetId())
+		if err != nil {
+			logger.Errorf("Error generating JWT Token: %v", err)
+		}
+		c.Request().Header.Add("Authorization", fmt.Sprintf("Bearer %s", token))
```

Comment:
We might want to cache the JWT token and generating a new token for every db request will add a lot of latency and CPU